lib/elixir/lib/supervisor.ex

```diff
@@ -131,8 +131,9 @@ defmodule Supervisor do
     * `:type` - specifies that the child process is a `:worker` or a
       `:supervisor`. This key is optional and defaults to `:worker`.
 
-  There is a sixth key, `:modules`, that is rarely changed. It is set
-  automatically based on the value in `:start`.
+
+    * `:modules` - this is rarely changed. This key is optional and it
+      is set automatically based on the `:start` value.
```

Comment:
We are still not explaining what this key does.

Reply:
Please keep it as before. The reason why we didn't set it on its own paragraph is to not highlight it. We can document it behaviour in another commit.

Reply:
please have a look at it again